src/webview2/mod.rs

```diff
@@ -646,6 +654,27 @@ impl InnerWebView {
       )?;
     }
 
+    webview.add_NewWindowRequested(
```

Comment:
This is the event I wanted to abuse to make html drag and drop and our file drop work at the same time but iirc it requires the dev to not use event.preventDefault() in the frontend so never really investigated much further.

Reply:
btw, did you commit this intentionally?

Reply:
Lol, nah, will remove that